Add Misty Theme.

etigerstudio opened the pull request:
Hi, I have started developing the Misty Theme for Typora in late 2017 and have been using it in Typora 
 personally for more than one year. I have been planning to share it with our fellow markdown lovers and Typora users ever since that time. Eventually, I uploaded it to my repo (https://github.com/etigerstudio/typora-misty-theme) on GitHub today and made this pull request.
Misty theme is simple yet beautiful theme. You may find detailed introduction in README of the repo (https://github.com/etigerstudio/typora-misty-theme/blob/master/README.md). 
Thank you!

etigerstudio committed:
Add Misty Theme.

abnerlee commented:
@etigerstudio Very nice theme.

Cloud you also include the thumbnails image file (in /media/thumbnails/misty-theme.png)? The image size is 5:4, such as 500x400px.
You could refer to https://github.com/typora/typora-theme-gallery/pull/60/files as an example

etigerstudio committed:
Copy Misty Theme thumbnail & preview images to /media.

etigerstudio committed:
Update 2019-01-11-Misty.md.

Upate front matter field ‘typora-copy-images-to’, same-domain image src, Windows-specific issue explanation.
Remove center alignment <p> wrapper.

etigerstudio commented:
Sorry, I forgot it. Now that's done. I hope I didn't made any other clumsy mistake this time. 🧐
